<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	border-radius: 5px;
	border: 1px solid #000000;
	padding: 10px;
}
QComboBox {
	border-radius: 5px;
	border: 1px solid #000000;
	padding: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>-1</y>
      <width>1121</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Xuất / Nhập kho</string>
     </attribute>
     <widget class="QTableWidget" name="prod_table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>861</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID hàng hóa</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tên hàng hóa</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Số lượng</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Đơn giá</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Đơn vị</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="add_prod_btn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>60</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Thêm mới</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_prod_btn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>120</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_btn">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Yêu cầu duyệt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_all_prod_btn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>180</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Xóa toàn bộ</string>
      </property>
     </widget>
     <widget class="QComboBox" name="prtn_box">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="type_box">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tổng</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_cost">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Đối tác</string>
     </attribute>
     <widget class="QTableWidget" name="prtn_table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>871</width>
        <height>261</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tên</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mô tả</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="update_prtn_btn">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Sửa đổi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_prtn_btn">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Thêm mới</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_prtn_btn">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>130</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Phiếu nhập kho</string>
     </attribute>
     <widget class="QTableWidget" name="import_table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>1031</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tên đối tác</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trạng thái</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tổng giá</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày tạo</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="im_order_item_btn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>300</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Chi tiết</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Phiếu xuất kho</string>
     </attribute>
     <widget class="QPushButton" name="ex_order_item_btn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>300</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Chi tiết</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="export_table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>1031</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tên đối tác</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trạng thái</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tổng giá</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày tạo</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Tài khoản</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tên đăng nhập</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_input">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Họ tên</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fullname_input">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_btn">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #b5de83;
	border-radius: 5px;
	border: 1px solid #000000;
	font-size: 16px
}</string>
      </property>
      <property name="text">
       <string>Đăng xuất</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
